public/store/view/index.ts

```diff
@@ -1,7 +1,9 @@
 export const LAST_STATE = '__LAST_STATE__';
 
 export interface ViewState {
+	fetchParamsCache: { [key: string]: string };
```

Comment:
You could use the `Dictionary<string>` type for this that can be imported from `util/dict.ts`.